CHANGE_LOG

```diff
@@ -1,3 +1,34 @@
+
+Version 0.21.0
+--------------
+
+This release introduces support for AMD's Heterogeneous System Architecture,
+which allows running the same code both on the CPU and the GPU.
```

Comment:
If you write Python code for HSA, won't it the same code able to be compiled for the CPU and the GPU? Or do they have different programming models still?